accounts/templates/register.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+{% block page_content %}
+
+
+            <div class="d-flex justify-content-center">
+                <div>
+                    <form method="POST" action="">
+                        {% csrf_token %}
+                            <div>{{form.username.label}} </div>
+                            <div>{{form.username}} </div>
+
+
+                            <div>{{form.email.label}}</div>
+                            <div>{{form.email}} </div>
+
+                            <div>{{form.password1.label}}</div>
+                            <div>{{form.password1}} </div>
+
+                            <div>{{form.password2.label}}</div>
+                            <div>{{form.password2}} </div>
+
+                            <div class="d-flex justify-content-center"><input type="submit" value="Zarejestruj się" name="Create user"> </div> 
+                    </form> 
+                </div>
+            </div>
+            <div style="text-align: center; margin-top: 20px;">
+                <a href="{% url 'login' %}">Masz już konto? Zaloguj się!!<a/>
+            </div>
+            <div class="d-flex justify-content-center">
+                {% for message in messages %}
+                    <p id=''>{{message}}</p>
+                {% endfor %}
+            </div>
+
+{% endblock %}
```

Comment:
Nie udało mi sie zarejestrować:
![image](https://user-images.githubusercontent.com/24852928/80431886-f8737200-88f2-11ea-8d3b-9f36522cbbe7.png)

Reply:
Django zapewnia wiele rodzajów walidacji. Jednym z nich jest "password is too common". I na pewno musi być dłuższe niż 8 liter. Póki co dla każdego z tych błędów wyświetlam "Wrong data has been passed"

Reply:
Skąd użytkownik ma wiedzieć co jest nie tak z danymi które próbuje wprowadzić?

Reply:
![Screenshot from 2020-04-28 10-33-38](https://user-images.githubusercontent.com/48382351/80465710-ca665000-893b-11ea-8cb6-631a04e52608.png)

Dodalem więcej szczegółowych informacji